website_menu_multilang/view/website.xml

```diff
@@ -1,26 +1,23 @@
 <?xml version="1.0" encoding="utf-8"?>
-<openerp>
-    <data>
+<odoo>
+  <!-- website.menu -->
+  <record id="menu_tree_inherit" model="ir.ui.view">
+    <field name="name">website.menu.tree.inherit</field>
+    <field name="model">website.menu</field>
+    <field name="inherit_id" ref="website.menu_tree"/>
+    <field name="arch" type="xml">
+      <field name="parent_id" position="after">
+        <field name="lang_ids" widget="many2many_tags"/>
+      </field>
+    </field>
+  </record>
 
-	<!-- website.menu -->
-        <record id="menu_tree_inherit" model="ir.ui.view">
```

Comment:
We all have likings about indentation, but please keep a smaller diff by respecting author's decision 😊